flocker/node/agents/_logging.py

```diff
@@ -94,3 +99,70 @@
     [VOLUME_ID, STATUS, TARGET_STATUS, WAIT_TIME],
     u"Waiting for a volume to reach target status.",)
 # End: Helper datastructures used by AWS storage driver.
+
+# Begin: Helper datastructures used by OpenStack storage drivers
+
+CODE = Field.for_types("code", [int], u"The HTTP response code.")
+MESSAGE = Field.for_types(
```

Comment:
Looks like this module has become a bit inconsistent about whether it uses `Field.for_types` or `Field.forTypes`.  Eliot is kind of weird in the way it tries to cater to both naming conventions.  We should stick to using just one, though.  And since we use `under_scores` most everywhere, I expect `Field.for_types` is the one to use.  So maybe update the `forTypes` uses near this code to be `for_types` instead.